pep-0678.rst

```diff
@@ -140,10 +140,10 @@ are collecting multiple exception objects to handle together. [1]_
 Specification
 =============
 
-``BaseException`` gains a new read-only attribute ``__notes__``, an initially
-empty tuple, and a new method ``.add_note(note: str)``.  ``note`` is added to
-the exception's notes, which appear in the standard traceback after the
-exception string. A ``TypeError`` is raised if ``note`` is not a string.
+``BaseException`` gains a new new method ``.add_note(note: str)``. Notes are
+exposed as a tuple via the read-only attribute ``__notes__``, which is backed
+by an internal list, and appear in the standard traceback after the exception
```

Comment:
I don't think the PEP needs to say what the internal implementation is.

Reply:
Fair enough, taken that back out - I've answered Brett on the SC issue anyway.